docs/messaging/azure-service-bus-component.md

```diff
@@ -180,12 +182,38 @@ The .NET Aspire Azure Service Bus component uses the following log categories:
 
 - `Azure.Core`
 - `Azure.Identity`
+- `Azure-Messaging-ServiceBus`
 
 ### Tracing
 
-The .NET Aspire Azure Service Bus component will emit the following tracing activities using OpenTelemetry:
-
-- "Azure.Data.Tables.TableServiceClient"
+To enable tracing, set the `DisableTracing` option to `false` in the configuration. When enabled, the .NET Aspire Azure Service Bus component will emit the following tracing activities using OpenTelemetry:
```

Comment:
> To enable tracing, set the `DisableTracing` option to `false` in the configuration.

This isn't correct. From the XML docs:

```
    /// <remarks>
    /// ServiceBus ActivitySource support in Azure SDK is experimental, the shape of Activities may change in the future without notice.
    /// It can be enabled by setting "Azure.Experimental.EnableActivitySource" <see cref="AppContext"/> switch to true.
    /// Or by setting "AZURE_EXPERIMENTAL_ENABLE_ACTIVITY_SOURCE" environment variable to "true".
    /// </remarks>
```